editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/util/maven/MavenUtils.java

```diff
@@ -299,12 +334,29 @@ public static boolean configureCamelVersionForProject(IProject project, String c
 	 * @return	true on success, otherwise false
 	 */
 	public static boolean configurePomCamelVersion(IProject project, CommonNewProjectMetaData projectMetaData, String camelVersion, IProgressMonitor monitor) {
-		String newCamelVersion = Strings.isBlank(camelVersion) && projectMetaData != null && !Strings.isBlank(projectMetaData.getCamelVersion()) ? projectMetaData.getCamelVersion() : camelVersion;
-		SubMonitor subMonitor = SubMonitor.convert(monitor,Messages.mavenTemplateConfiguratorAdaptingprojectToCamelVersionMonitorMessage, 8);
+		SubMonitor subMonitor = SubMonitor.convert(monitor,Messages.mavenTemplateConfiguratorAdaptingprojectToCamelVersionMonitorMessage, 7);
 		try {
 			File pomFile = new File(project.getFile(IMavenConstants.POM_FILE_NAME).getLocation().toOSString()); //$NON-NLS-1$
 			Model m2m = new CamelMavenUtils().getMavenModel(project);
-			subMonitor.setWorkRemaining(7);
+			
+			boolean ok = configurePomCamelVersion(project, m2m, projectMetaData, camelVersion, subMonitor.newChild(6));
```

Comment:
it is recommended to use subMonitor.split instead of subMonitor.newChild see 2.2 https://www.eclipse.org/articles/Article-Progress-Monitors/article.html

"There are several methods on SubMonitor which consume ticks. Namely, split(...), newChild(...), and worked(...). Practically speaking the only one you need is split(...).

I'll be using split(...) for most of the code examples in this article, but it is new in Eclipse 4.6. If your code is meant to work on earlier versions of Eclipse you should use newChild(...) instead of split(...). The two do pretty much the same thing except that split(...) also performs cancellation checks."

fine to do on a later PR